doc/sphinx/Pacemaker_Explained/cluster-options.rst

```diff
@@ -184,6 +184,16 @@ holds. So the decision was made to place them in an easy-to-find location.
      -
      - Node ID of the cluster's current designated controller (DC). Used and
        maintained by the cluster.
+   * - .. _execution_date:
+       
+       .. index::
+          pair: execution-date; cib
+       
+       execution-date
+     - :ref:`nonnegative integer <nonnegative_integer>`
```

Comment:
64-bit, not 32-bit, but best I can do without overhauling the option types.

Reply:
We need a "seconds since epoch" type (some other tables that haven't been converted to specify types yet will need it too)